Fatal error: exception Failure("illegal actual argument in call to foo: found Void, expected Bool in Call(bar, )")
